### ISSUE TITLE
UnusedUseStatement false positives

### ISSUE THREAD

**Issue by boesing** [webimpress/coding-standard#124]
Hey there,

I've found false-positives with `psalm-` or `phpstan-` prefixed annotations regarding `WebimpressCodingStandard.Namespaces.UnusedUseStatement`

```php

use IteratorAggregate;

class Foo
{
    /** @psalm-var IteratorAggregate */
    private $bar;
}
```

Expected: all good.
Actual:  
```
1 | ERROR | [x] Unused use statement "IteratorAggregate"
```

Same issue when passing more details to the `IteratorAggregate` annotation:

```php

use IteratorAggregate;

class Foo
{
    /** @var IteratorAggregate<string,string> */
    private $bar;

    /**
     * @param IteratorAggregate<string,string> $bar
     */
    public function __construct(ArrayObject $bar)
    {
        $this->bar = $bar;
    }
}
```

Expected: all good.
Actual:  
```
1 | ERROR | [x] Unused use statement "IteratorAggregate"
```

**Comment by michalbundyra** [comment]
@boesing thanks for reporting. This is pretty hard and I am not sure how I should handle it.

What I am thinking now is to use FQCN with custom PHPDocs tags... I think I could add support for psalm-/phpstan- tags, but then we might find other tools which need support here. This will is "a problem" with multiple sniffs in the library right now.
